lib/omnibus/build_version.rb

```diff
@@ -124,16 +124,16 @@ def semver
       build_tag
     end
 
-    # We'll attempt to retrive the timestamp from the Jenkin's set BUILD_ID
+    # We'll attempt to retrive the timestamp from the Jenkin's set BUILD_TIMESTAMP
```

Comment:
[nit] since you're touching that line, can you make it say "retrieve", please? 😃

Reply:
Sure